explainer.md

```diff
@@ -1,121 +1,159 @@
-    minRequiredNetwork: "network-non-mobile"  // default: "network-online"
-    allowOnBattery: true                      // default: true
-    idleRequired: false                       // default: false
-  }).then(function() {
-    // Success
-  }, function() {
-    // Failure - user/UA denied permission
-  });
+self.addEventListener('sync', function(event) {
+  if (event.registration.tag == 'outbox') {
+    event.waitUntil(sendEverythingInTheOutbox());
+  }
 });
 ```
-* `register` registers sync events for whichever SW matches the current document, even if it's not yet active.
-* `id`: Useful for recognizing distinct synchronization events. If the id is already registered, the old registration is replaced by the new one.
-* `minDelay`: The suggested number of milliseconds to wait before triggering the first sync event. This may be delayed further (for coalescing purposes or to reserve resources) by a UA-determined amount of time. Subsequent intervals will be based from the requested initial trigger time.
-* `maxDelay`: The suggested maximum number of milliseconds to wait before firing the event even if the conditions aren't met. In some resource constrained settings the maxDelayMs may be delayed further. Does not apply to periodic events. The default value is 0, which means no max.
-* `minPeriod`: A suggestion of the minimum time between sync events. A value of 0 (the default) means the event does not repeat. This value is a suggestion and may be delayed for a UA-specific period of time in resource constrained environments (e.g., when on battery). If the value is less than SyncManager.minAllowablePeriod (which is UA and platform dependent) then the promise will reject. Periodic sync registrations will repeat until the UA determines that they shouldn't anymore (e.g., the user doesn't visit the site frequently enough to merit the periodic sync). Because of this unpredictability, put critical functionality into non-periodic syncs or use push messaging.
-* `minRequiredNetwork`: One of "network-offline", "network-online", or "network-non-mobile".
-* `allowOnBattery`: False if the device must be on AC power when the event is fired.
-* `idleRequired`: True if the device must be in an idle state (UA determined) when the event is fired.
 
-## Handling Synchronization Events
+`sync` will fire when the UA believes the user has connectivity.
+
+The promise passed to `waitUntil` is a signal to the UA that the sync event is ongoing and that it should keep the SW alive if possible. Rejection of the event signals to the UA that the sync failed. Upon rejection the UA should reschedule (likely with a UA-determined backoff).
+
+The UA may coalesce syncronizations to reduce the number of times the device, radio and browser need to wake up. The coalescing can be across origins, or even OS. Although the event timings are coalesced, you still get an event per pending sync registration.
+
+## Periodic syncronization
```

Comment:
Hah yes, I was ignoring my spellchecker here because it was trying to change it to UK English